PhysicsTools/ONNXRuntime/interface/ONNXRuntime.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * ONNXRuntime.h
+ *
+ *  Created on: Jun 28, 2019
+ *      Author: hqu
+ */
+
+#ifndef PHYSICSTOOLS_ONNXRUNTIME_INTERFACE_ONNXRUNTIME_H_
+#define PHYSICSTOOLS_ONNXRUNTIME_INTERFACE_ONNXRUNTIME_H_
+
+#include <vector>
+#include <map>
+#include <string>
+#include <memory>
+
+#include "onnxruntime/core/session/onnxruntime_cxx_api.h"
+
+namespace Ort {
+
+  typedef std::vector<std::vector<float>> FloatArrays;
+
+  class ONNXRuntime {
```

Comment:
I understood from the links to ONNX repo that `Ort` is their namespace. Should we really put our code in that namespace? (wrt. one should put close to nothing to `std`)

Reply:
This `ONNXRuntime` class is basically a convenience wrapper of the ONNXRuntime C++ API,  so I think it's more convenient to keep the `Ort` namespace. Since `Ort` is the namespace of the exposed C++ API, not the internal onnxruntime implementation, it should not cause any clash with the library I think.